tests/snappi/test_snappi.py

```diff
@@ -124,6 +125,27 @@ def test_snappi(snappi_api,
     # """ Apply configuration """
     snappi_api.set_config(config)
 
+    # Check for ARP status to be resolved
```

Comment:
Since this feature will be used in every snappi script, can we use a function to implement this, e.g., wait_for_arp(snappi_api, max_attempts, poll_interval_sec) which returns actual # of attempts.

Reply:
implemented `wait_for_arp` as suggested